### ISSUE TITLE
Hello,I wnat to train my own database,please give some guidance.

### ISSUE THREAD

**Issue by ltc576935585** [ppengtang/pcl.pytorch#20]
I have alread make the selectivesearch data for my own database.when run the program,warning me I should have the  corresponding *.json.Please guide me how to make this for my database,I just have imgs and annotation like Pascal_voc form.

**Comment by ppengtang** [comment]
Hi, maybe you could follow codes provided [here](https://github.com/facebookresearch/Detectron/issues/204#issuecomment-374551705) to convert PASCAL format annotations to COCO format annotations.

**Comment by HMCCMH** [comment]
> I have alread make the selectivesearch data for my own database.when run the program,warning me I should have the corresponding *.json.Please guide me how to make this for my database,I just have imgs and annotation like Pascal_voc form.

Hello! I want to know whether you face the problem that the loss suddenly become low (maybe zero) when you use your private data?
